packages/ui/v2/core/WizardForm.tsx

```diff
@@ -12,7 +12,12 @@ type DefaultStep = {
   isLoading: boolean;
 };
 
-function WizardForm<T extends DefaultStep>(props: { href: string; steps: T[]; containerClassname?: string }) {
+function WizardForm<T extends DefaultStep>(props: {
+  href: string;
+  steps: T[];
+  disableNavigation?: boolean;
```

Comment:
Since the setup wizard is now two steps we want to hide the navigation buttons if a user navigates to the setup wizard when they're not supposed to.